libsol/message.c

```diff
@@ -99,6 +100,8 @@ int process_message_body(const uint8_t* message_body,
             case ProgramIdUnknown:
                 display_instruction_info[display_instruction_count++] = info;
                 break;
+            //Currently we don't display these instructions
+            case ProgramIdToken2022:
```

Comment:
At the moment it seems that the bug is here. We skip the logic where we count instructions and setting instruction info which leads us to unrecognized format handling upper in `handle_sign_message_parse_message` since `process_message_body` returns `1`.

Reply:
Hello, thanks for the tests and feedback @sponomarev. It is interesting that "that none of the basic operations" does not work, we've tested it on the real hardware too (NanoX and NanoSP) and it looked fine. 
I'll double check that this branch contains the correct code (as it was separated from the other feature recently), but I'm quite sure about that.

In addition to that, are you able to share the Transaction that you used to perform tests? It would be great to reproduce the problem with the same data that you have used. After that we will handle this problem asap.

Reply:
Thank you, yes, you are right, we will handle this and come back soon